test/regexmanager.cpp

```diff
@@ -227,8 +227,9 @@ TEST_CASE("quote_and_highlight wraps highlighted text in numbered tags",
 	}
 }
 
-TEST_CASE("RegexManager::dump_config turns each `highlight` and "
-	"`highlight-article` rule into a string, and appends them onto a vector",
+TEST_CASE("RegexManager::dump_config turns each `highlight`,"
+	"`highlight-article` and `highlight-feed` rule into a string, and appends"
+    "them onto a vector",
```

Comment:
Please put a space at the end of the string literal — when the literals are concatenated, no spaces are inserted, so we have to do it ourselves. In other words, `"hello"  "world"` becomes `"helloworld"`, so we do `"hello "  "world"` to get `"hello world"`.